querier/querier.go

```diff
@@ -13,7 +14,13 @@ import (
 	"github.com/weaveworks/cortex/util"
 )
 
-// NewQueryable creates a new promql.Engine for cortex.
+// NewEngine creates a new promql.Engine for cortex.
+func NewEngine(distributor Querier, chunkStore chunk.Store) *promql.Engine {
```

Comment:
Should remove `NewQueryable` function below and use `NewEngine` in `main.go`, no point in having both.